### ISSUE TITLE
feature request: git+generation integration for dotfile management

### ISSUE THREAD

**Issue by magnetophon** [MagneticDuck/nix-rice#2]
Great work you're doing, dotfile management is a major missing link in nixos, imho.

I know it's early days for this project, so see this as a brainstorm.

@matthiasbeyer is doing [related work](https://github.com/matthiasbeyer/nixos-scripts), he has [a script](https://github.com/matthiasbeyer/nixos-scripts/blob/master/nix-script-show-commit.sh) that shows you the nixpkgs commit that your current generation is derived from.
One of his goals is something similar [but for configuration](https://github.com/matthiasbeyer/nixos-scripts/issues/8).

I guess you can see where this is going:
Eventually, it'd be great to have a way to see which dotfile config each generation has.

**Comment by cgadski** [comment]
yeah, I'll write a helper script / user client for the file distribution real soon, first I'd like to get some real stuff working though. Been working on a bunch of stuff, including learning more about nixpkgs and nix, but this project is an end-point I'm dragging myself towards. Thanks for the interest!
